roles/dns/tasks/main.yml

```diff
@@ -27,3 +27,7 @@
   with_subelements:
   - "{{ dns_records_add | default({}) }}"
   - entries
+  register: nsupdate_result
+  until: nsupdate_result.dns_rc == 0
```

Comment:
What is dns_rc? Is it supposed to be `rc == 0`? Note, for loops, it may be better off using the `|success` pipe instead